### ISSUE TITLE
Removing and re-adding a glasskube repository with the same name doesn't update the URL

### ISSUE THREAD

**Issue by yeahservice** [glasskube/glasskube#1075]
**Describe the bug**
Removing and re-adding a glasskube repository with the same name doesn't update the URL

**To Reproduce**
```
# Add repository with invalid URL
glasskube repo add test3 https://raw.githubusercontent.com/yeahservice/packages
glasskube repo list

NAME          URL                                                                                                     DEFAULT  AUTHENTICATION  STATUS     MESSAGE
glasskube     https://packages.dl.glasskube.dev/packages                                                              Yes      None            Ready      repo has 20 packages                                                                                                   
test3         https://raw.githubusercontent.com/yeahservice/packages                                                  No       None            Not ready  failed to fetch https://raw.githubusercontent.com/yeahservice/packages/index.yaml: wrong status code: 400 Bad Request  

# Delete repository and re-add with valid URL
glasskube repo delete test3
glasskube repo add test3 https://raw.githubusercontent.com/yeahservice/packages/feature/nginx-ingress-namespace-scoped/packages
glasskube repo add test2 https://raw.githubusercontent.com/yeahservice/packages/feature/nginx-ingress-namespace-scoped/packages

# test3 repository still checks old URL, while test2 works as expected
glasskube repo list

NAME          URL                                                                                                     DEFAULT  AUTHENTICATION  STATUS     MESSAGE
glasskube     https://packages.dl.glasskube.dev/packages                                                              Yes      None            Ready      repo has 20 packages                                                                                                   
test2         https://raw.githubusercontent.com/yeahservice/packages/feature/nginx-ingress-namespace-scoped/packages  No       None            Ready      repo has 20 packages      
test3         https://raw.githubusercontent.com/yeahservice/packages/feature/nginx-ingress-namespace-scoped/packages  No       None            Not ready  failed to fetch https://raw.githubusercontent.com/yeahservice/packages/index.yaml: wrong status code: 400 Bad Request  

```

**Expected behavior**
After deleting and adding a repository with the same name but different URL, the URL should also be updated

**Screenshots**
-

**Cluster Info (please complete the following information):**
 - Type: Kubernetes
 - Version: v1.29.2
 - Provider: Kind

**Glasskube Info (please complete the following information):**
 - Package Operator version: v0.13.0
 - CLI version: glasskube version dev (commit 6c3e38ea7964b17efc7f5e09766067bec09cab34)

**Additional context**
-

**Comment by pmig** [comment]
Thanks for reporting! This bug was already fixed in https://github.com/glasskube/glasskube/pull/1054 and released in [v0.15.0](https://github.com/glasskube/glasskube/releases/tag/v0.15.0). Please make sure to update Glasskube to the latest version (v0.16.0).

**Comment by yeahservice** [comment]
My bad, I was checking the release notes before reporting it but apparently missed the fix. I can't reproduce it with v0.16.0. Closing.
